bin/travis-ci/deb_publish.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e 
+set -o pipefail
+
+# TO-DO
+echo "IMPLEMENT_ME"
```

Comment:
Should we include this file if it's noop?

Reply:
I'll be working on this next so it'll be replaced soon enough

Reply:
Sounds good